packages/grpc-js/src/subchannel-pool.ts

```diff
@@ -101,7 +101,7 @@ export class SubchannelPool {
       }, REF_CHECK_INTERVAL);
 
       // Unref because this timer should not keep the event loop running.
-      this.cleanupTimer.unref();
+      this.cleanupTimer.unref && this.cleanupTimer.unref();
```

Comment:
```suggestion
      // Call unref only if it exists to address electron/electron#21162
      this.cleanupTimer.unref?.();
```

I prefer this check, [introduced in Typescript 3.7](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html). I think it more directly communicates the intention of "call unref if it is present".

This will only work after #1184 is merged, or if this PR also makes the same upgrade of the `typescript` dependency as in that PR.